setup.py

```diff
@@ -9,16 +9,16 @@
 # If adding requirements make sure to also add to requirements-dev.txt
 install_requires = [
     'botocore>=1.8.0, <=1.8.21',
-    'aiohttp>=2.3.0, <3.0.0',
+    'aiohttp>=3.0.0',
     'multidict>=2.1.4',
     'wrapt>=1.10.10',
     'packaging>=16.8',
 ]
 
 PY_VER = sys.version_info
 
-if not PY_VER >= (3, 4, 1):
-    raise RuntimeError("aiobotocore doesn't support Python earlier than 3.4")
+if not PY_VER >= (3, 5, 0):
```

Comment:
3.5.3 actually. The same in all other places

Reply:
fixed, thanks